examples/tutorials/basics/polygons.py

```diff
@@ -0,0 +1,96 @@
+"""
+Plotting polygons
+=================
+
+Plotting polygons is handled by the :meth:`pygmt.Figure.plot` method.
+
+This tutorial focuses on input data given as NumPy arrays. Besides NumPy arrays,
+array-like objects are supported. For plotting a :class:`geopandas.GeoDataFrame` object
+with polygon geometries, e.g. to create a choropleth map, see the gallery example
+:doc:`Choropleth map </gallery/maps/choropleth_map>`.
+"""
+
+# %%
+import numpy as np
+import pygmt
+
+# %%
+# Plot polygons
+# -------------
+#
+# Set up sample data points as NumPy arrays for the x and y values.
+
+x = np.array([-2, 1, 3, 0, -4, -2])
+y = np.array([-3, -1, 1, 3, 2, -3])
+
+# %%
+# Create a Cartesian plot via the :meth:`pygmt.Figure.basemap` method. Pass arrays to
+# the ``x`` and ``y`` parameters of the :meth:`pygmt.Figure.plot` method. Without
+# further adjustments, lines are drawn between the data points. By default, the lines
+# are 0.25-points thick, black, and solid. In this example, the data points are chosen
+# to make the lines form a polygon.
+
+fig = pygmt.Figure()
+fig.basemap(region=[-5, 5, -5, 5], projection="X5c", frame=True)
+fig.plot(x=x, y=y)
+fig.show()
+
+# %%
+# The ``pen`` parameter can be used to adjust the lines or outline of the polygon.
+# The argument passed to ``pen`` is one string with the comma-separated optional
+# values *width*,\ *color*,\ *style*.
+
+fig = pygmt.Figure()
+fig.basemap(region=[-5, 5, -5, 5], projection="X5c", frame=True)
+# Use a 2-points thick, darkred, dashed outline
+fig.plot(x=x, y=y, pen="2p,darkred,dashed")
+fig.show()
+
+# %%
+# Use the ``fill`` parameter to fill the polygon with a color or
+# :doc:`pattern </techref/patterns>`.
+
+fig = pygmt.Figure()
+fig.basemap(region=[-5, 5, -5, 5], projection="X5c", frame=True)
+# Fill the polygon with color "orange"
+fig.plot(x=x, y=y, fill="orange", pen="2p,darkred,dashed")
+fig.show()
```

Comment:
I think it's better to keep three separate figures, but remove the `pen` parameter for the example explaining the `fill` parameter to show that now lines are drawn in case `fill` is used. Please see commit https://github.com/GenericMappingTools/pygmt/pull/3593/commits/7463a822d729851caf19d881ef7f65bcecef6b95.